python/src/aiconfig/editor/client/src/components/prompt/PromptMenuButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Menu, createStyles } from "@mantine/core";
+import { IconDotsVertical, IconTrash } from "@tabler/icons-react";
+import { memo } from "react";
+
+const useStyles = createStyles((theme) => ({
+  cellAction: {
+    marginLeft: -8,
+    [theme.fn.smallerThan("sm")]: {
+      float: "left",
+      marginTop: -2,
```

Comment:
Feels kind of strange to use negative margins (and kind of related to my question about why not put this within the prompt container itself), but yea not a big deal